integration/e2e/service.go

```diff
@@ -235,8 +241,17 @@ func (s *ConcreteService) Ready() error {
 	return s.readiness.Ready(s)
 }
 
-func (s *ConcreteService) containerName() string {
-	return NetworkContainerHost(s.usedNetworkName, s.name)
+func (s *ConcreteService) setContainerName() error {
+	s.containerName = NetworkContainerHost(s.usedNetworkName, s.name)
+	// Linux supports up to 64 chars for hostname
+	if len(s.containerName) > 64 {
+		return fmt.Errorf("service container name %s should not exceed 64.", s.containerName)
```

Comment:
Nit: for the error message I would put the offending argument within "[]", so something like "service container name [%s] should not exceed 64."
 
`service container name long long longstring should not exceed 64.`  it's not as easy to be parsed by eyeball as `service container name [long long longstring] should not exceed 64.`

Also I think Go's convention is to not end error message with any punctuation according to https://github.com/golang/go/wiki/CodeReviewComments#error-strings?

Reply:
Sorry for late response. Thanks for your great comment about the guideline of error message. I will update soon.